benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_operations.ipynb

```diff
@@ -164,21 +164,22 @@
    ]
   },
   {
-   "cell_type": "raw",
-   "id": "dac6e619-cfbf-46ad-9d54-6ba3317bcc97",
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "263e09d2-0fbe-4b5e-8e4c-aa682a14b6f1",
    "metadata": {
     "tags": []
    },
+   "outputs": [],
    "source": [
     "# Update input variables\n",
     "start_datetime = elasticsearch_widgets.get_start_datetime()\n",
     "end_datetime = elasticsearch_widgets.get_end_datetime()\n",
     "elastic_index = elasticsearch_widgets.get_elastic_index()\n",
     "# Caution: Enable it in case of delete\n",
-    "## delete_result = elasticsearch.delete_index_ids_between_dates(index=elastic_index,start_datetime=start_datetime, end_datetime=end_datetime)\n",
-    "logging.warning(f\"index '{elastic_index}': deleted\")\n",
     "ids_list = elasticsearch.get_index_ids_between_dates(index=elastic_index,start_datetime=start_datetime, end_datetime=end_datetime)\n",
-    "logging.warning(f\"index '{elastic_index}': {len(ids_list)} documents\")"
+    "delete_result = elasticsearch.delete_index_ids_between_dates(index=elastic_index,start_datetime=start_datetime, end_datetime=end_datetime)\n",
```

Comment:
Is this actually enabling deletion of something that was previously not deleted?

You may want to sanitize the start and end times so you don't inadvertently wipe everything out.

Reply:
You right, fix it